### ISSUE TITLE
Question for training

### ISSUE THREAD

**Issue by Daisygotit666** [YuYue525/MobileSAM-pytorch#8]

**Comment by Daisygotit666** [comment]
Hello, I found that there is just Tiny_Vit model in your train code,  does it means you just funetune the image encoder?not the whole mobile-sam model including prompt encoder and mask decoder besides image encoder.

**Comment by YuYue525** [comment]
You are right.

**Comment by Sencc** [comment]
Hi,I also have the same question, how do you funetune the mask encoder?

**Comment by Daisygotit666** [comment]
> Hi,I also have the same question, how do you funetune the mask decoder?

Actually, its easy to finetune mask decoder, I can share you a website, maybe you can make it.https://colab.research.google.com/drive/1F6uRommb3GswcRlPZWpkAQRMVNdVH7Ww?usp=sharing

**Comment by Sencc** [comment]
@Daisygotit666 thank u! I got it.
